### DISCUSSION TITLE
Adding a bird

### DISCUSSION THREAD

**Discussion by tmceld** [mcguirepr89/BirdNET-Pi#907] [Q&A]
Really loving birdnetpi - really great project - i have it running off a car battery attached to a solar panel - its great.

One of the reasons i set this up was i wanted to track the spread of ring-neck parakeets which are increasingly common here in Manchester, UK https://www.manchestereveningnews.co.uk/news/greater-manchester-news/many-parakeets-south-manchester-wild-15642144

Now, i understand why they aren't being detected, as i have my location in, but is there anyway i can add the bird so it can be detected?

**Comment by ehpersonal38** [comment]
See the discussion here, also read the comments before you start editing code as there are some corrections: https://github.com/mcguirepr89/BirdNET-Pi/discussions/841#discussioncomment-5573066
